### PR TITLE
Feature/398 static fe

### PR DESCRIPTION
**Pull request by bobbi-hbrown**
https://trello.com/c/0xR0uMZQ/398-implement-front-end-checkbox-elements-for-filters?filter=label:Tech

### Change description
Implements the components needed for the /download page. There are no tests currently, but these can be implemented as we work on connecting the filters to the backend.

Some extra details are needed (specified in the TODOs) but are not considered a priority at the moment.

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

File: app/templates/main/logout.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{% macro logout_partial(sso_logout_url, get_help_url) %}
+<p class="flex-parent-element flexed-element-margins-collapse govuk-!-text-align-right">
+    <a href="{{ get_help_url }}"
+       class="govuk-link govuk-link--no-underline govuk-!-margin-right-5">
+        Get help</a>
+    <a href="{{ sso_logout_url }}" class="govuk-link govuk-link--no-underline">
+        Log out
+    </a>
+</p>
+{% endmacro %}
```
Comment (gidsg) [inline app/templates/main/logout.html:10]: Do we still want this partial if we're not using it yet? Maybe leave for a separate PR specific for Log Out/Get Help?

**Review by gidsg — CHANGES_REQUESTED** [review]
` govuk_frontend.zip` should not be checked in. Please can we remove.

**Review by DonShelly — APPROVED** [review]

**Comment by gidsg** [comment]
@bobbi-hbrown the  `govuk_frontend.zip` file is still checked in. Please remove it, i.e.
```
git rm govuk_frontend.zip
git commit
```

**Comment by bobbi-hbrown** [comment]
pre-commit.ci autofix

**Review by gidsg — APPROVED** [review]

**Comment by gidsg** [comment]
@bobbi-hbrown LGTM 👍 but please squash and merge as commit history is quite messy.
